### ISSUE TITLE
NavigationCompleted  connectionaborted 

### ISSUE THREAD

**Issue by 17lcc** [MicrosoftEdge/WebView2Feedback#3538]
[](url)I had this problem. For the win10 computer with the same configuration, one could load the page, while the other could not. The error message of connectionaborted on the NavigationCompleted was displayed. The webview2 plugin is up to
![Uploading 4E6107ED-3426-4250-8044-A9D1DDCFB86F.png…]()
 date

**Comment by 17lcc** [comment]
webview2 initialization does not report an error. What is the reason

**Comment by champnic** [comment]
@17lcc Are you saying that the same exact app/code running on two different machines exhibited different behavior? Or are you using different apps? A sample app or code may help figure out what's going on here. Or a network trace:
https://github.com/MicrosoftEdge/WebView2Feedback/blob/main/diagnostics/network.md#network-logs

**Comment by 17lcc** [comment]
Yes, I have the exact same application running on two different machines, exhibiting different behavior, the same web page doesn't load on one of them

**Comment by 17lcc** [comment]
![Uploading 1686022259210_1ABC36FA-EEDA-4b94-ABE3-C9F33CAD40B7.png…]()

**Comment by champnic** [comment]
FYI- your images don't seem to be finishing uploading. I can't access them. Want to try again?
